toolbox/fixit/main.go

```diff
@@ -0,0 +1,231 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	u "istio.io/test-infra/toolbox/util"
+)
+
+type orderV string
+type sortV string
+
+const (
+	// Allowed field values can be found at https://developer.github.com/v3/search/#search-repositories
+	desc    = orderV("desc")
+	created = sortV("created")
+)
+
+var (
+	org             = flag.String("user", "istio", "Github owner or org")
+	tokenFile       = flag.String("token_file", "/Users/hklai/github_token", "Github token file (optional)")
+	repo            = flag.String("repo", "istio", "Github repo")
+	label            = flag.String("label", "kind/fixit", "Label to search for")
+	sort            = flag.String("sort", string(created), "The sort field. Can be comments, created, or updated.")
+	order           = flag.String("order", string(desc), "The sort order if sort parameter is provided. One of asc or desc.")
+	startDate       = time.Date(2018, time.March, 5, 0, 0, 0, 0, time.Local)
+	endDate       = time.Date(2018, time.March, 10, 0, 0, 0, 0, time.Local)
+	gh              *u.GithubClient
+)
+
+func init() {
+	flag.Parse()
+	if *tokenFile != "" {
+		token, err := u.GetAPITokenFromFile(*tokenFile)
+		if err != nil {
+			log.Fatalf("Error accessing user supplied token_file: %v\n", err)
+		}
+		gh = u.NewGithubClient(*org, token)
+	} else {
+		gh = u.NewGithubClientNoAuth(*org)
+	}
+}
+
+// Increment a key in the given map.
+func incrementCount(m map[string]int, key string) {
```

Comment:
replace code in function with  
```
m[key]++ 
```
or get rid of function entirely

++ works and default return value is zero so it works (tested)